pkg/server/io/helpers_windows.go

```diff
@@ -68,6 +68,16 @@ func (dc *delayedConnection) Close() error {
 	return nil
 }
 
+func newEmptyPipes() (_ *stdioPipes, _ *wgCloser, _ error) {
+	ctx, cancel := context.WithCancel(context.Background())
+	return &stdioPipes{}, &wgCloser{
+		wg:     &sync.WaitGroup{},
+		set:    make([]io.Closer, 0),
+		ctx:    ctx,
+		cancel: cancel,
+	}, nil
+}
+
```

Comment:
Two things:
1. This should be moved out of a windows specific file as there's no `newEmptyPipes` definition for Unix platforms. As this is called in `pkg/server/io/container_io.go` which doesn't have a build restraint of any sort this will make it such that cri will only be able to compile on Windows. 
2. Unless I'm missing something and there is some definition for newEmptyPipes somewhere, there's no reason to return an error for this call as nothing can fail.

Reply:
makes sense